cluster/v1alpha1/0000_03_clusters.open-cluster-management.io_placements.crd.yaml

```diff
@@ -12,7 +12,17 @@ spec:
   scope: Namespaced
   preserveUnknownFields: false
   versions:
-  - name: v1alpha1
+  - additionalPrinterColumns:
+    - jsonPath: .status.conditions[?(@.type=="PlacementSatisfied")].status
+      name: Succeeded
+      type: string
+    - jsonPath: .status.conditions[?(@.type=="PlacementSatisfied")].reason
+      name: Reason
+      type: string
+    - jsonPath: .status.numberOfSelectedClusters
+      name: SelectedClusters
+      type: integer
```

Comment:
I think you can add print columns by adding kubebuilder markers instead of updating the generated CRD directly.
Here is an example: https://github.com/open-cluster-management-io/api/blob/9667ae902e4b505bbcecd8eedfebcc2281718f3b/addon/v1alpha1/types_clustermanagementaddon.go#L12-L13

Reply:
@elgnay Not sure if it's my kubebuilder version issue, the sequence changes. This part is the same as https://github.com/open-cluster-management-io/api/pull/93/files#diff-8584af8df06b8cc38c1d152b48e5c590f870be9b792b6abb178c66f89201c41fL265.